custom/enikshay/management/commands/soft_delete_cases_with_no_person.py

```diff
@@ -0,0 +1,51 @@
+import csv
+
+from django.core.management import BaseCommand
+
+from corehq.form_processor.interfaces.dbaccessors import CaseAccessors
+
+from custom.enikshay.case_utils import get_person_case
+from custom.enikshay.exceptions import ENikshayCaseNotFound
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument('domain')
+        parser.add_argument('case_type')
+        parser.add_argument('log_file_name')
+        parser.add_argument('--commit', action='store_true')
+        parser.add_argument('--deletion_id', dest='deletion_id', default='case_without_person')
+
+    def handle(self, domain, case_type, log_file_name, **options):
+        commit = options['commit']
+        deletion_id = options['deletion_id']
+
+        with open(log_file_name, 'w') as log_file:
+            logger = self.get_logger(log_file)
+            for case_id in self.get_case_ids(domain, case_type):
```

Comment:
would you want to use a `with_progress_bar` here?

Reply:
added